# Service for homing (Resetting the offset) all finger joints
---
bool success
